networks/filecoin/devTool.csv

```diff
@@ -1,10 +1,23 @@
 slug,provider,toolType,tag,price,description,starred,actionButtons
-filecoin-signing-tools,!provider:filecoin-signing-tools,,,,,,"[""[Docs](https://docs.filecoin.io/reference/built-in-actors)""]"
-fvm-assemblyscript-sdk,!provider:fvm-assemblyscript-sdk,,,,,,"[""[GitHub](https://github.com/Zondax/fvm-as-sdk)""]"
-fvm-tinygo-sdk,!provider:fvm-tinygo-sdk,,,,,,"[""[GitHub](https://github.com/ipfs-force-community/go-fvm-sdk)""]"
-powergate-textile,!provider:powergate-textile,,,,,,"[""[Docs](https://docs.textile.io/powergate/)"",""[GitHub](https://github.com/textileio/powergate)""]"
-tableland-db,!provider:tableland-db,,,,,,"[""[Website](https://tableland.xyz/)""]"
-orbit-db,!provider:orbit-db,,,,,,"[""[Website](https://orbitdb.org/)""]"
-thread-db,!provider:thread-db,,,,,,"[""[Website](https://docs.textile.io/threads/)""]"
-go-graphsplit,!provider:go-graphsplit,,,,,,"[""[GitHub](https://github.com/filedrive-team/go-graphsplit)""]"
-beryx-address-converter,!provider:beryx-address-converter,,,,,,"[""[Website](https://beryx.io/address_converter)""]"
+filecoin-signing-tools,Filecoin Signing Tools,,,,,,"[""[Docs](https://docs.filecoin.io/reference/built-in-actors)""]"
```

Comment:
If you want to migrate record to the network level - it has to be migrated fully, like it is done in Astar gas station in this PR, and it has to be removed from Provider level.